### DISCUSSION TITLE
How to turn off the workflow description in the chat process when using new-api to access the dify workflow

### DISCUSSION THREAD

**Discussion by JZhouKai** [open-webui/open-webui#7946] [Q&A]
Hi everyone,
    I don not konw how to turn off the workflow description in the chat process when using new-api to access the dify workflow. Thank you for your answers!
<img width="851" alt="1734594217196" src="https://github.com/user-attachments/assets/f5e8c32b-65b2-4dd2-933a-f49322d007be" />

**Comment by batman0911** [comment]
same question, I create a workflow and I want to integrate it to my app without node description in api response

**Reply by JZhouKai** [reply]
If there are any ideas, we can communicate together.
